src/cl/lodestar/lodestar_launcher.star

```diff
@@ -22,6 +21,22 @@ DISCOVERY_PORT_NUM			= 9000
 HTTP_PORT_NUM				= 4000
 METRICS_PORT_NUM			= 8008
 
+# The min/max CPU/memory that the beacon node can use
+BEACON_MIN_CPU = 50
+BEACON_MAX_CPU = 1000
+BEACON_MIN_MEMORY = 256
+BEACON_MAX_MEMORY = 1024
+
+#  ---------------------------------- Validator client -------------------------------------
+VALIDATING_REWARDS_ACCOUNT	= package_io.VALIDATING_REWARDS_ACCOUNT
```

Comment:
nit: can just use `package_io.VALIDATING_REWARDS_ACCOUNT` downstream where ever you use `VALIDATING_REWARDS_ACCOUNT`

Reply:
fixed